src/Application/Actions/Donations/Create.php

```diff
@@ -84,6 +92,14 @@ protected function action(): Response
             $donation = $this->donationRepository->buildFromApiRequest($donationData);
```

Comment:
@NoelLH This line seems redundant. Isn't it already in the `try` block above?

Reply:
@alihejazi It's a re-try, as noted in the comment. If it failed the first time then `$donation` is undefined. This is also not changing in this PR.